src/Tests/Microsoft.NET.Build.Tests/GivenFrameworkReferences.cs

```diff
@@ -192,6 +193,78 @@ public void TargetingPackDownloadCanBeDisabled()
                 .HaveStdOutContaining("NETSDK1073");
         }
 
+        [CoreMSBuildOnlyTheory]
```

Comment:
```suggestion
        [Theory]
```

I'm not seeing why this requires .NET Core msbuild. I think it should pass fine when using .NET Framework msbuild as well. Copied from above?